core/server/api/shared/headers.js

```diff
@@ -31,6 +31,24 @@ const disposition = {
             'Content-Type': 'application/yaml',
             'Content-Length': JSON.stringify(result).length
         };
+    },
+
+    /**
+     * ### Content Disposition Header
+     * create a header that invokes the 'Save As' dialog in the browser when exporting the database to file. The 'filename'
+     * parameter is governed by [RFC6266](http://tools.ietf.org/html/rfc6266#section-4.3).
+     *
+     * For encoding whitespace and non-ISO-8859-1 characters, you MUST use the "filename*=" attribute, NOT "filename=".
+     * Ideally, both. Examples: http://tools.ietf.org/html/rfc6266#section-5
+     *
+     * We'll use ISO-8859-1 characters here to keep it simple.
+     *
+     * @see http://tools.ietf.org/html/rfc598
+     */
+    file(result) {
```

Comment:
Needed for https://github.com/TryGhost/Ghost/pull/10051/files#diff-5aec6ddaa47f7b2024e989d0cf8af73fR44